src/taipy/rest/api/resources/cycle.py

```diff
@@ -69,9 +75,9 @@ class CycleResource(Resource):
                 properties:
                   msg:
                     type: string
-                    example: cycle deleted
+                    example: Cycle deleted
```

Comment:
What is that 'example'??
Should this be a 'description' instead?
I mean: what is 'msg', at the end of the day?

Reply:
msg is message and that's it. We don't need to have a description for that, and that is an example of what the message can possibly be.

Reply:
"We don't need to have a description for that"? What do you mean we don't need that?
We are producing some output. You may think the 'msg' wording is self-explanatory. I don't.
And what's the use of the example? Who can this be useful for?

Reply:
So the example should be removed. And the description should be "Error message" or something similar right?

Reply:
Right